### PR TITLE
feat：数据分析服务下，选择时间段逻辑优化 #7326

### PR DESCRIPTION
**Pull request by yjieliang**
feat：数据分析服务下，选择时间段逻辑优化 #7326

### CODE REVIEW

File: src/backend/ci/core/metrics/biz-metrics/src/main/kotlin/com/tencent/devops/metrics/utils/QueryParamCheckUtil.kt [hunk 1]
```diff
@@ -82,17 +82,20 @@ object QueryParamCheckUtil {
...
-        // 查询时间区间限制，当天的前一天至前六个月
+        // 目前仅支持6个月内的数据查询
         val startDate = DateTimeUtil.stringToLocalDate(startTime)
         val endDate = DateTimeUtil.stringToLocalDate(endTime)
-        val firstDate = LocalDate.now()
-        val secondDate = firstDate.minusMonths(metricsConfig.maximumQueryMonths)
-        if (startDate!!.isBefore(secondDate)) {
+        if ((startDate!!.until(endDate, ChronoUnit.DAYS)) > metricsConfig.queryDaysMax) {
             throw ErrorCodeException(
                 errorCode = MetricsMessageCode.QUERY_DATE_BEYOND
```
Comment (carlyin0801) [inline src/backend/ci/core/metrics/biz-metrics/src/main/kotlin/com/tencent/devops/metrics/utils/QueryParamCheckUtil.kt:90]: 错误描述有问题，错误提示信息不应该写死，应该搞成根据配置文件的配置搞成动态化

Comment (yjieliang) [reply src/backend/ci/core/metrics/biz-metrics/src/main/kotlin/com/tencent/devops/metrics/utils/QueryParamCheckUtil.kt]: 已调整

**Review by carlyin0801 — APPROVED** [review]
ok

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-ci?pullRequest=7355) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-ci?pullRequest=7355) before we can accept your contribution.<br/>**6** out of **7** committers have signed the CLA.<br/><br/>:white_check_mark: yongyiduan<br/>:white_check_mark: sawyersong2<br/>:white_check_mark: mingshewhe<br/>:white_check_mark: yjieliang<br/>:white_check_mark: carlyin0801<br/>:white_check_mark: irwinsun<br/>:x: devops<br/><hr/>**devops** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-ci?pullRequest=7355) it.</sub>

**Pull request closed without merging**